test/electricity/circuits/waveguides/maximum_electric_field_intensity_of_main_wave_in_rectangular_waveguide_test.py

```diff
@@ -6,7 +6,7 @@
 
 ## The relative permittivity of the material filling the waveguide is 2.2. The width of the cross-section
 ## of the waveguide is 4 centimeter, the wavelength is 10 millimeter. The magnetic field strength is 100 ampere
-## per meter. Then the maximum electric field strength will be 1694 millivolt per meter.
+## per meter. Then the maximum electric field strength will be 203 kilovolt per meter.
```

Comment:
Why did it change so drastically? Is it because there were wrong units in vacuum impedance?